ePBFT/src/pbft/libbyz/test/test_ledger_replay.cpp

```diff
@@ -83,31 +84,32 @@ void init_replica(std::vector<char>& service_mem)
 
 TEST_CASE("Test Ledger Replay")
 {
+  ringbuffer::Circuit eio(2);
+  auto wf = ringbuffer::WriterFactory(eio);
+
```

Comment:
FYI using the `WriterFactory` (now that I have un-split `ledger.h`) adds 5 seconds to this test. Not sure why since I am not using the ringbuffer methods, but the simple file read/write methods from the `Ledger` class

Reply:
This may only be in debug - it doesn't seem to have made the test any slower on the CI